samples/DirectX/D3D12/HelloConstBuffer12.cs

```diff
@@ -70,9 +70,9 @@ public unsafe class HelloConstBuffer12 : DX12Sample
         // App resources.
         private ID3D12Resource* _vertexBuffer;
         private D3D12_VERTEX_BUFFER_VIEW _vertexBufferView;
-        private ID3D12Resource* _constantBuffer;
+        private readonly ID3D12Resource* _constantBuffer;
         private SceneConstantBuffer _constantBufferData;
-        private byte* _pCbvDataBegin;
+        private readonly byte* _pCbvDataBegin;
```

Comment:
This isn't `readonly`, it is mutated by `unsafe` code.

Reply:
correct